packages/server-wallet/src/db/migrations/20200915140300_chain_service.ts

```diff
@@ -0,0 +1,19 @@
+import * as Knex from 'knex';
+
+const channels = 'channels';
+const chainServiceRequests = 'chain_service_requests';
+const fundingStrategy = 'funding_strategy';
+
+export async function up(knex: Knex): Promise<any> {
+  await knex.schema.alterTable(channels, table => {
+    table.specificType(chainServiceRequests, 'text[]').notNullable();
+    table.string(fundingStrategy).notNullable();
```

Comment:
I think we will have to edit this migration so that it can run in existing wallets.

As is, the migration will fail if there are any existing rows in the `channels` table, since this migration
1. requires a non-null `funding_strategy` value
2. does not provide a default value

If we provided a default value, then we would have to either
A. take down existing wallets for maintenance; or
B. accept that existing wallets will experience a pause while this migration runs, considering that it will require a write-lock on the `channels` table.

Reply:
In the future, wallets may want to upgrade in a rolling fashion, so they can still serve requests during the upgrade. At that point, we can explore more advanced migration strategies (they are not that hard, they just require care).